chia/server/node_discovery.py

```diff
@@ -208,20 +208,21 @@ async def _query_dns(self, dns_address):
             if self.resolver is None:
                 self.log.warn("Skipping DNS query: asyncresolver not initialized.")
                 return
-            peers: List[TimestampedPeerInfo] = []
-            result = await self.resolver.resolve(qname=dns_address, lifetime=30)
-            for ip in result:
-                peers.append(
-                    TimestampedPeerInfo(
-                        ip.to_text(),
-                        self.default_port,
-                        0,
+            for rdtype in ["A", "AAAA"]:
+                peers: List[TimestampedPeerInfo] = []
+                result = await self.resolver.resolve(qname=dns_address, rdtype=rdtype, lifetime=30)
+                for ip in result:
+                    peers.append(
+                        TimestampedPeerInfo(
+                            ip.to_text(),
+                            self.default_port,
+                            0,
+                        )
                     )
-                )
-            self.log.info(f"Received {len(peers)} peers from DNS seeder.")
-            if len(peers) == 0:
-                return
-            await self._respond_peers_common(full_node_protocol.RespondPeers(peers), None, False)
+                self.log.info(f"Received {len(peers)} peers from DNS seeder, using rdtype = {rdtype}.")
+                if len(peers) == 0:
+                    continue
+                await self._respond_peers_common(full_node_protocol.RespondPeers(peers), None, False)
```

Comment:
this looks a bit more convoluted than I think it needs to be. wouldn't it be more straight-forward to say:
```
if len(peers) > 0:
   await self._respond_peers_common(full_node_protocol.RespondPeers(peers), None, False)
```